src/app/loginComponents/accounts/external/accounts.component.html

```diff
@@ -14,120 +14,131 @@
-      <span class="pull-right" fxLayout="column" fxLayoutAlign="end end">
-        <button
-          *ngIf="row.isLinked === false; else linked"
-          mat-raised-button
-          color="secondary"
-          class="pull-right"
-          [attr.id]="'link-' + row.name"
-          (click)="link(row.source)"
-          matTooltip="Link the account"
-        >
-          <mat-icon *ngIf="row.name !== 'ORCID'; else orcid">link</mat-icon>
-          <span> Create or connect your {{ row.name }} Account</span>
-        </button>
-      </span>
-      <ng-template #linked>
-        <button
-          class="mb-2"
-          mat-raised-button
-          color="primary"
-          [attr.id]="'relink-' + row.name"
-          (click)="relink(row.source)"
-          matTooltip="Relink the account"
-        >
-          <mat-icon *ngIf="row.name !== 'ORCID'; else orcid">link</mat-icon>
-          <span> Relink {{ row.name }} Account</span>
-        </button>
-        <button
-          mat-raised-button
-          color="primary"
-          [attr.id]="'unlink-' + row.name"
-          (click)="unlink(row.source)"
-          matTooltip="Unlink the account"
-        >
-          <mat-icon *ngIf="row.name !== 'ORCID'; else orcid">link_off</mat-icon>
-          <span> Unlink {{ row.name }} Account</span>
-        </button>
-      </ng-template>
-      <ng-template #orcid>
-        <img src="../../assets/images/account-logos/{{ row.logo }}" alt="ORCID iD logo" class="orcid-id-logo site-icons-small" />
-      </ng-template>
-    </div>
-  </mat-card>
+  <div *ngFor="let row of accountsInfo" class="p-3" fxFlex="50" fxFlex.lt-md="100">
+    <mat-card class="mb-2 card-style">
+      <div class="h-100" fxLayout="column" fxLayoutAlign="space-between stretch">
+        <div>
+          <div fxLayoutAlign="space-between">
+            <mat-card-header fxLayoutAlign="flex-start" fxLayoutGap="1.5rem">
+              <div mat-card-avatar>
+                <div *ngIf="row.isLinked" matBadge="✓" matBadgeColor="green"></div>
+                <img src="../../../../assets/images/account-logos/{{ row.logo }}" alt="{{ row.name }} logo" class="account-logo" />
+              </div>
+              <mat-card-title class="title">{{ row.name }} Account&nbsp;</mat-card-title>
+              <mat-card-subtitle class="color" *ngIf="row.isLinked; else placeholder">
+                <strong>Username: {{ row.source | getTokenUsername: tokens }} </strong>
+                <span *ngIf="row.name === 'ORCID'">
+                  <strong> ID: </strong>
+                  <a href="https://orcid.org/{{ orcidId$ | async }}"> {{ orcidId$ | async }}</a>
+                </span>
+              </mat-card-subtitle>
+              <ng-template #placeholder><br /></ng-template>
+            </mat-card-header>
+            <mat-chip-list>
+              <mat-chip class="gray-label-tag" *ngIf="row.control" matTooltip="TBD"> Source Control </mat-chip>
```

Comment:
Tooltip here and lines 96 and 97, should not be `TBD`

Reply:
Sorry, I forgot to mention this in my comment but I wasn't sure what to put for the tooltip for these so I temporarily put TBD. I wanted to ask you guys for ideas on what to put for those since these are newly added into this component!

Reply:
I feel like they're self explanatory except for the Research chip.

Reply:
Maybe `Pull your workflows from cloud source control to make them available on Dockstore`